edx_proctoring/templates/timed_exam/submitted.html

```diff
@@ -14,7 +14,7 @@
   <hr>
   <p>
     {% blocktrans %}
-      As you have submitted your exam, you will not be able to re-enter it. Your grades for this timed exam will be immediately available on the <a href="{{progress_page_url}}">Progress</a> page.
+      Because you have submitted your exam, you can no longer access the exam. Your grade for this timed exam is available on the <a href="{{progress_page_url}}">Progress</a> page.
```

Comment:
Is it available immediately? If not, and we don't want to say "will be", perhaps something like "Check your *Progress* page for your grade for this timed exam."